tests/system/Language/LanguageTest.php

```diff
@@ -126,6 +127,28 @@ public function testGetLineArrayFormatsMessages(): void
         $this->assertSame(['45 related books.'], $this->lang->getLine('books.bookList', [91 / 2]));
     }
 
+    /**
+     * @see https://github.com/codeigniter4/shield/issues/851
+     */
+    public function testGetLineInvalidFormatMessage(): void
+    {
+        // No intl extension? then we can't test this - go away....
+        if (! class_exists(MessageFormatter::class)) {
+            $this->markTestSkipped('No intl support.');
+        }
+
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
+            'Invalid message format: "تم الكشف عن كلمة المرور {0} بسبب اختراق البيانات وشوهدت {1 ، عدد} مرة في {2} في كلمات المرور المخترقة.", args: "password,hits,wording"'
```

Comment:
Is this normal (language) for tests?

Reply:
Yes it might be better to make a comment as to explain testing of Arabic as international languages.

Reply:
Set locale to `ar`.